sklearn/manifold/tests/test_mds.py

```diff
@@ -59,3 +61,63 @@ def test_MDS():
                     [4, 2, 1, 0]])
     mds_clf = mds.MDS(metric=False, n_jobs=3, dissimilarity="precomputed")
     mds_clf.fit(sim)
+
+
+def test_svd_mds():
+    # Generate 4 randomly chosen points
+    Y = np.array([[1,  0,  1],
+                  [-1, 3,  2],
+                  [1, -2,  3],
+                  [2, -1, -3]])
+    sim = euclidean_distances(Y)
+    # calculate error or smacof-based solution
+    X_smacof, smacof_stress = mds.smacof(sim, n_components=2, random_state=42)
+    X_svd, svd_stress = mds.svd_mds(sim, n_components=2)
+    assert_less(svd_stress, smacof_stress)
```

Comment:
Actually, smacof tries to solve a not convex problem using convex optimization technics. There is no guarantee to find the optimum (while there is using the eigenvector strategy).